app/src/main/java/com/libraryapp/Utilities/RESTController.java

```diff
@@ -1,4 +1,93 @@
 package com.libraryapp.Utilities;
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.ProtocolException;
+import java.net.URL;
 
 public class RESTController {
-}
+
+    private static OutputStream outputStream;
+    private static BufferedWriter bufferedWriter;
+
+    public static String sendGet(String urlGet) throws IOException {
+        URL url = new URL(urlGet);
```

Comment:
Maybe the parameter `urlGet` could be changed to `url` and the url on line 13 could be something like `requestUrl`